src/app/authentication/forgot-password.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm }    from '@angular/forms';
+import { AuthService } from '../services/auth.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+
+@Component({
+    selector: 'forgot-password',
+    templateUrl: './forgot-password.component.html',
+    styleUrls: ['./login.component.scss']
+})
+export class ForgotPasswordComponent {
+
+    resetForm : FormGroup;
+    errorMessage : String;
+    successMessage : String;
+
+    constructor(private _authService: AuthService, fb: FormBuilder) {
+        this.resetForm = fb.group({
+            'email' : [null, Validators.compose([Validators.required, Validators.pattern("[^ @]*@[^ @]*")])]
+        });
+    }
+
+    onSubmit(form: any): void {
+        this._authService.resetPassword(form).subscribe(res => {
+            if(res.info){
+                this.errorMessage = res.info;
+                this.successMessage = "";
```

Comment:
i'm not very familiar with typescripts, but i'm sure there are linting rules available for it as well (a quick googling gave me [this](https://github.com/palantir/tslint)).
try to keep typescript consistent as well (quotes, spacing etc)